api-catalog-ui/frontend/src/components/Wizard/configs/wizard_base_categories.jsx

```diff
@@ -94,18 +104,21 @@ export const baseCategories = [
                 question: 'The relative path to the home page of the service:',
                 optional: true,
                 regexRestriction: [wizRegex.validRelativeUrl],
+                tooltip: 'e.g. /application/home',
```

Comment:
Add additional information, something like: `Normally used for informational purposes for other services to use it as a landing page, example: /home`